package.json

```diff
@@ -77,35 +77,32 @@
 		"nodeArguments": [
 			"--loader=ts-node/esm",
 			"--experimental-specifier-resolution=node"
-		],
-		"nonSemVerExperiments": {
-			"configurableModuleFormat": true
-		}
+		]
 	},
 	"devDependencies": {
 		"@rollup/plugin-json": "^4.1.0",
-		"@rollup/plugin-node-resolve": "^13.0.6",
-		"@rollup/plugin-typescript": "^8.3.0",
+		"@rollup/plugin-node-resolve": "^13.3.0",
+		"@rollup/plugin-typescript": "^8.3.3",
 		"@sindresorhus/tsconfig": "^2.0.0",
-		"@types/estree": "^0.0.50",
-		"@types/jsdom": "^16.2.13",
-		"ava": "^3.15.0",
+		"@types/estree": "^0.0.52",
+		"@types/jsdom": "^16.2.14",
+		"ava": "^4.3.0",
 		"github-reserved-names": "^2.0.4",
-		"jsdom": "^19.0.0",
+		"jsdom": "^20.0.0",
 		"npm-run-all": "^4.1.5",
 		"parcel-bundler": "^1.12.4",
 		"parcel-plugin-svelte": "^4.0.9",
-		"rollup": "^2.60.2",
+		"rollup": "^2.75.7",
 		"rollup-plugin-terser": "^7.0.2",
 		"strip-indent": "^4.0.0",
-		"svelte": "^3.44.2",
-		"svelte-check": "^2.2.10",
-		"ts-node": "^10.4.0",
-		"tslib": "^2.3.1",
-		"typescript": "^4.5.2",
-		"xo": "^0.47.0"
+		"svelte": "^3.49.0",
+		"svelte-check": "^2.8.0",
+		"ts-node": "^10.8.2",
+		"tslib": "^2.4.0",
+		"typescript": "^4.7.4",
+		"xo": "^0.50.0"
 	},
 	"engines": {
-		"node": ">=12"
+		"node": ">=14"
```

Comment:
is this a breaking change?

Reply:
Maybe. I think npm will output a warning (varies by npm version). 

I don’t think it’s worth the time to test this so let’s undo it for now.

Reply:
How about… 🤭 

- https://github.com/refined-github/refined-github/pull/5801

Reply:
Hmm maybe will try it out

Reply:
> Maybe. I think npm will output a warning (varies by npm version).

> I don’t think it’s worth the time to test this so let’s undo it for now.

Really we needed to update a long time ago since we are using optional chaining